typescript/examples/model-context-protocol-smart-wallet-server/index.ts

```diff
@@ -57,13 +57,24 @@ async function initializeServer() {
     );
 
     // Configure CDP Wallet Provider
+    const privateKey = (process.env.PRIVATE_KEY || generatePrivateKey()) as Hex;
+    const signer = privateKeyToAccount(privateKey);
+
     const config = {
       apiKeyName: process.env.CDP_API_KEY_NAME!,
       apiKeyPrivateKey: process.env.CDP_API_KEY_PRIVATE_KEY!,
       networkId: process.env.NETWORK_ID || "base-sepolia",
+      smartWalletAddress: process.env.SMART_WALLET_ADDRESS! as Address,
```

Comment:
SMART_WALLET_ADDRESS is optional, should we get the address from the generated key in the case that it's not provided?

Reply:
I'll remove the `!`, as its unnecessary. 

If SMART_WALLET_ADDRESS is not provided, one will be generated, so no concerns there. In that case, we export the generated wallet address from the provider and log it below this code